### PR TITLE
correct path to `htmlSafe` and `isHTMLSafe` in docs

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: packages/ember-glimmer/lib/utils/string.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function htmlSafe(str: string) {
   Detects if a string was decorated using `htmlSafe`.
 
   ```javascript
-  import { htmlSafe, isHTMLSafe } from '@ember/string';
+  import { htmlSafe, isHTMLSafe } from '@ember/tempalte';
```
Comment (boris-petrov) [inline packages/ember-glimmer/lib/utils/string.ts]: You have a typo here. :)

**Comment by turino2018** [comment]
I don’t know how to do that lol

Sent from my iPhone

On 15 Aug 2018, at 20:53, Boris Petrov <notifications@github.com> wrote:

@boris-petrov commented on this pull request.

In packages/ember-glimmer/lib/utils/string.ts:

> @@ -88,7 +88,7 @@ export function htmlSafe(str: string) {
   Detects if a string was decorated using `htmlSafe`.
 
   ```javascript
-  import { htmlSafe, isHTMLSafe } from '@ember/string';
+  import { htmlSafe, isHTMLSafe } from '@ember/tempalte';
You have a typo here. :)

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.

File: packages/ember-glimmer/lib/utils/string.ts (hunk 1 above)
Comment (bekzod) [reply packages/ember-glimmer/lib/utils/string.ts]: thanks

**Review by Turbo87 — APPROVED** [review]

**Comment by bekzod** [comment]
@locks maybe we can merge this 😉

**Review by rwjblue — APPROVED** [review]

**Pull request merged**